ts-tests/tests/test-gas-limit.ts

```diff
@@ -11,24 +10,24 @@ import { BodhiSigner } from "@acala-network/bodhi";
 describeWithAcala("Acala RPC (GasLimit)", (context) => {
 	let alice: BodhiSigner;
 
-    before(async () => {
+    beforeAll(async () => {
         [alice] = context.wallets;
     });
 
     it("block gas limit", async () => {
         const contract = await deployContract(alice, Factory);
         // limited by used_storage
-        const result = await contract.createContractLoop(350);
-        expect(result.gasLimit.toNumber()).to.be.eq(3569798122);
+        const result = await contract.createContractLoop(200);
```

Comment:
not sure why this is reduced so much. can you check? @zjb0807

Reply:
#2795 fix this